modules/sdpolicy-networking.adoc

```diff
@@ -22,12 +26,16 @@ To use a custom hostname for a route, you must update your DNS provider by creat
+* Internal control load balancer that is internal to the cluster and used to balance traffic for internal cluster communications.
+* External control load balancer that is used for accessing the OpenShift and Kubernetes APIs. This load balancer can be disabled in {OCM}. If this load balancer is disabled, Red Hat reconfigures the API DNS to point to the internal control load balancer.
+* External control load balancer for Red Hat that is reserved for cluster management by Red Hat. Access is strictly controlled, and communication is only possible from whitelisted bastion hosts.
+* Default router/ingress load balancer that is the default application load balancer, denoted by `apps` in the URL. The default load balancer can be configured in OCM to be either publicly accessible over the Internet, or only privately accessible over a pre-existing private connection. All application routes on the cluster are exposed on this default router load balancer, including cluster services such as the logging UI, metrics API, and registry.
 * Optional: Secondary router/ingress load balancer that is a secondary application load balancer, denoted by `apps2` in the URL. The secondary load balancer can be configured in OCM to be either publicly accessible over the Internet, or only privately accessible over a pre-existing private connection. If a 'Label match' is configured for this router load balancer, then only application routes matching this label will be exposed on this router load balancer, otherwise all application routes are also exposed on this router load balancer.
-* Optional: Load balancers for services can also be purchased to enable non-HTTP/SNI traffic and non-standard ports for services.  These load balancers can be mapped to a service running on {product-title} to enable advanced ingress features, such as non-HTTP/SNI traffic or the use of non-standard ports. These can be purchased in groups of 4 for standard clusters or can be provisioned without charge in CCS clusters, however each AWS account has a quota which link:https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-limits.html[limits the number of Classic Load Balancers] that can be used within each cluster.
+* Optional: Load balancers for services that can can be mapped to a service running on OSD to enable advanced ingress features, such as non-HTTP/SNI traffic or the use of non-standard ports. These can be purchased in groups of 4 for standard clusters or can be provisioned without charge in Customer Cloud Subscription (CCS) clusters, however each AWS account has a quota which link:https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-limits.html[limits the number of Classic Load Balancers] that can be used within each cluster.
```

Comment:
- s/OSD/{product-title}
- s/clusters or can be/clusters, or they can be
- s/clusters, however each/clusters; however, each/
- s/has a quota which/has a quota that